packages/client/src/StreamrClient.ts

```diff
@@ -192,6 +191,12 @@ export class StreamrClient extends EventEmitter { // eslint-disable-line no-rede
     // TODO annotate connection parameter as internal parameter if possible?
     constructor(options: StreamrClientOptions = {}, connection?: StreamrConnection) {
         super()
+
+        if (typeof uid === 'undefined') {
+            // use process id in node uid
+            uid = process.pid != null ? `${process.pid}` : `${uuid().slice(-4)}${uuid().slice(0, 4)}`
+        }
```

Comment:
Please add some comments as to why this is done here otherwise it's likely it'll end up accidentally refactored back to how it was

Reply:
Good point.